astropy/stats/bayesian_blocks.py

```diff
@@ -286,7 +286,7 @@ def fitness(self, **kwargs):
     def p0_prior(self, N):
         """
         Empirical prior, parametrized by the false alarm probability ``p0``
```

Comment:
Needs a full stop here (or semicolon and lower case `see`)

Reply:
✅